curp/src/server/curp_node.rs

```diff
@@ -47,6 +48,9 @@ pub(super) enum CurpError {
     /// Encode or decode error
     #[error("encode or decode error")]
     EncodeDecode(#[from] bincode::Error),
+    /// Protobuf Encode or decode error
```

Comment:
Who will receive this `CurpError`? And is it need to tell apart the `EncodeDecode` Error and the `PbDecode` Error?

Reply:
The `CurpError` will be converted to tonic::Status which indicate an internal error. I merge these two error in to one `EncodeDecode` error